media/js/firefox/sticky-promo.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var StickyPromo = {};
+    var stickyPromoCookieID = 'firefoxStickyPromo';
```

Comment:
I'm setting the cookie ID one place, rather than passing args through each function. Should I put this variable in ALL CAPS?

Reply:
I'm not sure we have an established convention for this. I guess we can look for other examples and see what we've done before.

Reply:
Looks like elsewhere we've done lowercase with dashes, like `firefox-sticky-promo`

Reply:
Revised.